### ISSUE TITLE
pysam does not work under SL6.6

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/galaxy#1662]
```
  File "/usr/local/galaxy/galaxy-dist/lib/galaxy/datatypes/tabular.py", line 18, in <module>
    from galaxy.datatypes.sniff import get_headers
  File "/usr/local/galaxy/galaxy-dist/lib/galaxy/datatypes/sniff.py", line 19, in <module>
    from galaxy.datatypes.binary import Binary
  File "/usr/local/galaxy/galaxy-dist/lib/galaxy/datatypes/binary.py", line 13, in <module>
    import pysam
  File "/usr/local/galaxy/galaxy-dist/.venv/lib/python2.7/site-packages/pysam/__init__.py", line 1, in <module>
    from pysam.libchtslib import *
ImportError: /usr/local/galaxy/galaxy-dist/.venv/lib/python2.7/site-packages/pysam/libchtslib.so: undefined symbol: gzopen64
```

Seems that zlib is missing.

**Comment by dannon** [comment]
@bgruening this is the old wheel, or the one in https://github.com/galaxyproject/galaxy/pull/1638 ?

**Comment by bgruening** [comment]
This is with 16.01 so I guess the `old` one.

**Comment by dannon** [comment]
Ok.  Could you test the new one in #1638?  I'm downloading SL, but it'll take a while.

**Comment by bgruening** [comment]
Same problem. It can be solved by sourcing the toolshed zlib package. So I guess you need to include this object as well.

**Comment by natefoo** [comment]
@bgruening Could you post the contents of your `/etc/os-release` please?

**Comment by bgruening** [comment]
[galaxy@cn029 galaxy-dist]$ cat /etc/redhat-release 
Scientific Linux release 6.6 (Carbon)

**Comment by natefoo** [comment]
Additional discussion on #1638 that probably should have been here. `common_startup.sh` needs to generate a `binary-compatibility.cfg` for SL6 as we do for CentOS.

**Comment by dannon** [comment]
Resolved with #1669
